inception-doc/src/main/resources/META-INF/asciidoc/user-guide/workload.adoc

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020
+// Ubiquitous Knowledge Processing (UKP) Lab
+// Technische Universität Darmstadt
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[sect_workload]]
+= Workload
+
+NOTE: This functionality is only available when the experimental feature dynamic-workload is enabled.
+
+Similar to the monitoring page, the workload page shall give project manager a fast overview of all documents
```

Comment:
"... the workload page gives the project manager..."

Reply:
It also allows you to configure an automatic distribution of documents among users, so please add this to the introduction sentence.